packages/InteractiveProfilingTool-Core.package/IPTTreeModel.class/instance/getSourceCodeFromMessage..st

```diff
@@ -0,0 +1,16 @@
+code browser
+getSourceCodeFromMessage: aMessageTally
+
+	| source class selector|
+	
+	(self checkIfClassExists: aMessageTally)
+	ifFalse: [^ 'Missing'].
+	
+	class := self getClass: aMessageTally.
+	selector := aMessageTally method selector.
+	source := (class == UndefinedObject and: [selector == #DoIt]) 
+		ifTrue: ['DoIt', String cr, String cr, 'Missing']
```

Comment:
What does this String indicate? Also please move it to a function. (Storing magic values in functions is a Squeak best-practice.)

Reply:
Bei `DoIt` handelt es sich um eine spezielle Methode in Squeak. Squeak für während der Laufzeit diese Methode ins Methoden-Directory ein, um Codefragmente auszuführen. Sind diese ausgeführt wird die Methode wieder gelöscht. Da es sich nach Absprache mit Eva um diese besondere Methode handelt sollen wir diese gesondert behandeln. 
Zudem denke ich das hier nicht zwangsläufig eine Methode nötig ist, da dieser Fall nur einmal vorkommt und dies aus dem Code ersichtlich ist.

Reply:
Ich glaube wir haben ein grundwätzlich anderes Verständnis davon, was `aus dem Code ersichtlich` bedeutet. Für mich heißt das, dass ich schon beim ersten Lesen eine recht klare Idee habe, was der Code tut. Ich möchte mich außerdem nicht 1 min lang auf eine Zeitle konzentrieren zu müssen um zu verstehen wozu sie da ist.